README.rst

```diff
@@ -72,15 +72,16 @@ Community
 
 The wasp-os community is centred around the
 `github project <https://github.com/wasp-os/wasp-os>`_ and is
-supplemented with instant messaging via the #wasp-os IRC channel at
-libera.chat .
-
-If you are unfamiliar with IRC and don't have a preferred client then
-we recommend connecting to libera.chat using the
-`matrix/IRC bridge <https://app.element.io/#/room/#wasp-os:libera.chat>`_.
-The matrix bridge will allow us to receive messages whilst offline. Follow
-the link above and, if you do not already have a matrix account, register
-yourself. That should be enough to get you chatting!
+supplemented with instant messaging in the
+`#wasp-os:matrix.org <https://app.element.io/#/room/#wasp-os:matrix.org>`_
```

Comment:
```suggestion
`#wasp-os:matrix.org <https://matrix.to/#/%23wasp-os%3Amatrix.org>`_
```

this allows people to choose their preferred client

Reply:
Thanks. I have split the difference and provided both links... I'd still like to provide a strong recommendation on client for those who have never touched Matrix and element is (IMHO) a fairly sane default!